src/main/drivers/light_ws2811strip.c

```diff
@@ -37,6 +37,7 @@
 #include "common/color.h"
 #include "common/colorconversion.h"
 #include "dma.h"
+#include "io.h"
```

Comment:
IMO this is overoptimization. This is not C++, we are not including megabytes of header files ...